client/components/google-my-business/stats-nudge.js

```diff
@@ -0,0 +1,51 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal Dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import SectionHeader from 'components/section-header';
+
+export const StatsNudge = ( { translate } ) => {
+	return (
+		<div>
+			<SectionHeader
+				className="google-my-business__stats-nudge-header"
+				label={ translate( 'We recommend Google My Business' ) }
+				href="#"
+			/>
+			<Card className="google-my-business__stats-nudge">
+				<div className="google-my-business__stats-nudge-image-wrapper">
+					<img
+						className="google-my-business__stats-nudge-image"
+						src="/calypso/images/google-my-business/phone-screenshot.png"
+						alt="Your business with Google My Business"
+					/>
+				</div>
+				<div className="google-my-business__stats-nudge-info">
+					<h1 className="google-my-business__stats-nudge-heading">
+						{ translate( 'Is your business listed on Google?' ) }
+					</h1>
+					<h2 className="google-my-business__stats-nudge-description">
+						{ translate(
+							'Google My Business is a fast and free way to drive traffic to your local business.'
+						) }
+					</h2>
+					<div className="google-my-business__stats-nudge-button-row">
+						<Button primary>{ translate( 'Drive More Traffic' ) }</Button>
```

Comment:
Hi! I've found a possible matching string that has already been translated **2** times:
`translate( 'Drive traffic to your blog' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).